src/lib/editor-routes.js

```diff
@@ -0,0 +1,20 @@
+import { Route, Redirect } from 'react-router';
+import React from 'react';
+import AppController from 'components/editor/AppController';
+import ArticleListController from 'components/editor/ArticleListController';
+import ArticleController from 'components/editor/ArticleController';
+import AuthorListController from 'components/editor/AuthorListController';
+import AuthorController from 'components/editor/AuthorController';
+
+export default (
+  <Route path="/" component={AppController}>
+    <Route path="articles/page/:page/" component={ArticleListController}>
+      <Route path="article/slug/:slug/" component={ArticleController} />
+      <Redirect to="article/slug/:slug" from="article/slug/:slug/" />
+    </Route>
+    <Redirect from="articles/page/:page" to="articles/page/:page/" />
+    <Route path="authors/" component={AuthorListController}>
+      <Route path="author/slug/:slug" component={AuthorController} />
```

Comment:
There's also no need to put the word slug here. Unless you have other ways of accessing the author, just author/:slug is fine.